From aa5f68559efb0fa94385462ed05af7153479334e Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Mon, 25 May 2020 08:58:03 +0300
Subject: [PATCH] [SW-0] goya2: asic: fix compilation error

As goya2 register offsets can be accessed through
header file goya2_blocks_linux_driver.h, pll base
offset define mmPSOC_PCI_PLL_BASE has been replaced
by mmPSOC_PCI_PLL_CTRL_BASE. This patch corrects
resulting compilation error.

Change-Id: I73278cffbf6c0f98236aeba7cbc6d11cfd07cdd2
---

diff --git a/goya2/asic.c b/goya2/asic.c
index 64c857b..37f3477 100644
--- a/goya2/asic.c
+++ b/goya2/asic.c
@@ -25,13 +25,14 @@
 u64 hl_pll_timestamp_freq_get(void)
 {
 	/* As per Goya2 PLL cfg.xslx, PCI PLL Div[3] drives CPU timestamp */
-	return hl_pll_get_freq(mmPSOC_PCI_PLL_BASE, HL_PLL_DIV3);
+	return hl_pll_get_freq(mmPSOC_PCI_PLL_CTRL_BASE, HL_PLL_DIV3);
 }
 
 void hl_uart_init(void)
 {
 	/* As per Goya2 PLL cfg.xlsx, PCI PLL Div[3] drives uart */
-	u32 uart_clk_freq = hl_pll_get_freq(mmPSOC_PCI_PLL_BASE, HL_PLL_DIV3);
+	u32 uart_clk_freq = hl_pll_get_freq(mmPSOC_PCI_PLL_CTRL_BASE,
+					    HL_PLL_DIV3);
 
 	preboot_uart_init(uart_clk_freq / HL_UART_BAUDRATE);
 }
@@ -40,7 +41,8 @@
 {
 	u32 factor;
 	/* As per Goya2 PLL cfg.xlsx, PCI PLL Div[3] drives SPI */
-	u32 spi_clk_freq = hl_pll_get_freq(mmPSOC_PCI_PLL_BASE, HL_PLL_DIV3);
+	u32 spi_clk_freq = hl_pll_get_freq(mmPSOC_PCI_PLL_CTRL_BASE,
+					   HL_PLL_DIV3);
 
 	factor = spi_clk_freq / HL_SPI_FLASH_MAX_FREQ;
 
